python/src/aiconfig/scripts/rage/rage.py

```diff
@@ -0,0 +1,60 @@
+import os
+import lastmile_utils.lib.core.api as core_utils
+import sys
+import time
+import itertools
+import numpy as np
+from result import Ok, Result
+
+logger = core_utils.get_logger(__name__)
+
+
+class RageConfig(core_utils.Record):
+    log_level: str | int = "WARNING"
+
+
+def rage(config: RageConfig) -> Result[None, str]:
+    logger.setLevel(config.log_level)
+    print("\n\n\n\n............\n\n")
+    print("Please open an issue! :) Here are some tips on how to do that:")
+    for logfile in ["editor_flask_server.log", "aiconfig.log"]:
```

Comment:
Why do we need to do two logfiles? I think this could be confusing to users having to submit 2 separate logfiles + issues

Reply:
Would be better to merge together

Reply:
Yeah this is just laziness / it's a P1 to write the server logs to aiconfig.log. Once we do that, we can just use the one log file. I think it's acceptable for a first cut